jolly-jellyfish/src/page_maker/templates/page_maker/template_detail.html

```diff
@@ -0,0 +1,16 @@
+{% extends "page_maker/base.html" %}
+
+{% block header %}
+<link rel="stylesheet" type="text/css" href="{{ template.style_sheet.url }}">
+{% endblock %}
+
+{% block content %}
+<div><h1>{{ template.name }}</h1></div>
+<!-- <img src=""> -->
```

Comment:
What's with these commented out images?

Reply:
I was supposed to put there some placeholder images for template preview but I forgot.